### PR TITLE
PWN-8245 - show rates loading warning in snackbar how it was before

### PR DESCRIPTION
**Pull request by eduardmaximovich**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-8245

## Description of Work

Reverted the snackbar error that occurs when unable to load token rates.
Originally, it was accidentially removed down there [link](https://github.com/p2p-org/key-app-android/pull/1547/files#diff-7d20ada08a280cc7d4172bc771d6c4a73602cb105ebd010f0f0aba4425fcc229L404)

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt [hunk 1]
```diff
@@ -132,7 +132,14 @@ class HomePresenter(
                 view?.showRefreshing(true)
                 userInteractor.loadAllTokensDataIfEmpty()
                 val solTokens = userInteractor.loadUserTokensAndUpdateLocal(tokenKeyProvider.publicKey.toPublicKey())
-                async { userInteractor.loadUserRates(solTokens) }
+                async {
+                    try {
+                        userInteractor.loadUserRates(solTokens)
+                    } catch (t: Throwable) {
+                        Timber.e(t, "Error on loading user rates")
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt:139]: no need to Timber.e this stuff, only Timber.i

**Review by gslevinkov — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Comment by gslevinkov** [comment]
no task number in the description

**Comment by gslevinkov** [comment]
let's put this fix in 2.7.1

File: app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt (hunk 1 above)
Comment (Davran331122) [reply app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt:139]: try catch will not handle error without using `await()`

**Review by Davran331122 — CHANGES_REQUESTED** [review]

**Review by Davran331122 — APPROVED** [review]

**Pull request merged**